svelte/demo/src/app/samples/toast/Action.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import {Slot} from '@agnos-ui/svelte/slot';
+	import type {ToastSlots} from '@agnos-ui/svelte/components/toast';
+	import biCheckCircleFill from 'bootstrap-icons/icons/check-circle-fill.svg?raw';
+	import biArrowCounterClockwise from 'bootstrap-icons/icons/arrow-counterclockwise.svg?raw';
+
+	// cf https://github.com/ota-meshi/eslint-plugin-svelte/issues/348
+	type $$Props = ToastSlots['structure']; // eslint-disable-line @typescript-eslint/no-unused-vars
+	type $$Slots = ToastSlots; // eslint-disable-line @typescript-eslint/no-unused-vars
+
+	export let state: $$Props['state'];
+	export let widget: $$Props['widget'];
+
+	function actionDemo() {
+		window.alert('Undo');
+	}
+</script>
+
+<div class="d-flex w-100">
+	<div class="d-flex align-items-center flex-grow-1 toast-body">
+		<!-- eslint-disable-next-line svelte/no-at-html-tags -->
```

Comment:
Once https://github.com/AmadeusITGroup/AgnosUI/pull/478 is merged, you can remove the eslint-disable comments.